fastlane-plugin-toolkit_s3.gemspec

```diff
@@ -17,8 +17,5 @@ Gem::Specification.new do |spec|
 	spec.metadata['rubygems_mfa_required'] = 'true'
 	spec.required_ruby_version = '>= 2.6'
 
-	# Don't add a dependency to fastlane or fastlane_re
-	# since this would cause a circular dependency
-
-	# spec.add_dependency 'your-dependency', '~> 1.0.0'
```

Comment:
Do you no longer need this documentation here for added dependencies?

Reply:
i dont think we need a reminder here to not add a circular dependency to fastlane